habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py

```diff
@@ -112,6 +113,22 @@ def __init__(
         self.prev_orientation = None
         self.walk_mocap_frame = 0
 
+        ## Load hand data
```

Comment:
I assume we eventually want humanoid arm animation in habitat-lab for ML training (perhaps enabled/disabled via a config). So this code should eventually go into habitat-lab `main`/`SIRo`.